parse.go

```diff
@@ -370,11 +374,16 @@ func cmdFromStruct(name string, dest path, t reflect.Type) (*command, error) {
 			case key == "help": // deprecated
 				spec.help = value
 			case key == "env":
+				pfx := envPrefix
+				if pfx != "" {
+					pfx += "_"
```

Comment:
Probably better to not add an underscore -- I bet there will be some need for a prefix without an underscore

Reply:
I implemented it this way following the example from [Viper](https://github.com/spf13/viper?tab=readme-ov-file#working-with-environment-variables) (as suggested by the linked issue).

Viper does use `_` as a separator between the prefix and the environment variable names:
https://github.com/spf13/viper/blob/3f40d9a463ca6305283d803796eb4b9fb72e5e66/viper.go#L412-L418

Users coming from Viper will probably find it more natural if go-arg has the same behaviour.

Either way, let me know what you prefer. It's an easy change anyway! :)

Reply:
Understood, but I do think it would be better not to add the underscore. I'm certain the no-underscore need will arise eventually. A lot of people will probably make the mistake of omitting the underscore when they wanted one, but when they look at --help they will see right away what's going on.

Reply:
> A lot of people will probably make the mistake of omitting the underscore when they wanted one, but when they look at --help they will see right away what's going on.

Ah yes, indeed it would be self-explanatory. I agree!